packages/components/src/components/hive-navigation/index.stories.tsx

```diff
@@ -71,6 +72,15 @@ export const Default: StoryObj = {
   ],
   args: {
     developerMenu: HIVE_DEVELOPER_MENU,
+    children: (
+      <VersionDropdown
+        currentVersion="v1"
+        versions={[
+          { label: 'v1', href: '#v1' },
+          { label: 'v2', href: '#v2' },
```

Comment:
```suggestion
          { label: 'Hive v1 Docs', href: '#v1' },
          { label: 'Hive v2 Docs', href: '#v2' },
```
similar to 
<img width="371" alt="image" src="https://github.com/user-attachments/assets/910f20e1-dad0-45c2-bd4e-7431e8317e20" />

(I want to see how it looks like in strorybook)